azurerm/internal/services/bot/parse/bot_channel_email.go

```diff
@@ -0,0 +1,33 @@
+package parse
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type BotChannelEmailId struct {
+	ResourceGroup string
+	BotName       string
```

Comment:
this also wants a field for Name:

```suggestion
	BotName       string
	Name             string
```

Reply:
Oh dangit! Should have gone with my first mind. Thx!